docker/scripts/demoLibrary.source

```diff
@@ -17,11 +17,11 @@ startDemoWrapper() {
     if [ "$1" == "seven" ]; then
         startDemo "$1" "mysql-openemr" "/etc/php/7.2/apache2" "ubuntu" "18-04" "18.04" "/var/www/html" "7"
     elif [ "$1" == "eight" ]; then
-        startDemo "$1" "mysql-openemr" "/etc/php7" "alpine" "3-12" "3.12" "/var/www/localhost/htdocs" "4"
+        startDemo "$1" "mysql-openemr" "/etc/php7" "alpine" "3-12" "3.12" "/var/www/localhost/htdocs" "8"
```

Comment:
wondering about `--env "DOCKERNUMBERDEMOS=${8}" `
should that match `$1`

Reply:
whoops
`# 8th parameter is to set to how many demos it will contain (can select a number from 1 - 10)`